src/main/java/com/amihaiemil/camel/RtYamlLines.java

```diff
@@ -8,10 +8,8 @@
 /**
  * YamlLines default implementation.
  * @author Mihai Andronache (amihaiemil@gmail.com)
- * @version $Id$
+ * @version $Id: aa086c8b637f198e26bc2ab0550fff6763e30016 $
```

Comment:
@SherifWaly This id should not be committed, it's automatically generated by git. Can you remove it and leave just ``$Id$``?